kedro/framework/hooks/manager.py

```diff
@@ -73,6 +73,7 @@ def _register_hooks_setuptools(
     plugin_names = set()
     disabled_plugin_names = set()
     for plugin, dist in plugininfo:
+        print("DEBUG!!!", plugin, dist)
```

Comment:
I'm guessing you only added this for debugging? 😄

Reply:
Thanks! 😅 I was working between different branches must have left this accidentally.